routers/matchmaking.js

```diff
@@ -194,6 +191,10 @@ var GlobalRoomInstances = Object.create(null);
 
 setInterval(CleanupInstances, 300 * 1000);
 
+if (require('../config.json').debug_trace_instances) {
```

Comment:
Useful debugging tool - logs instance data every 30 seconds if there is at least 1 instance open and the `debug_trace_instances` config variable is truthy.